binary_search/binarySearch.py

```diff
@@ -0,0 +1,28 @@
+#Binary Search in python3
+#Define an array with elements
```

Comment:
add proper spacing between operators and operands.

Reply:
Okay.. I will do

Reply:
Do I need to remove those first two lines ? @t2013anurag

Reply:
Ahh, no. Keep them.